packages/core/docs/general/performance.md

```diff
@@ -0,0 +1,49 @@
+# Performance
```

Comment:
in the (near) future we should have a big docs about this module that perf doc will be a part of :)

Reply:
That's a good idea 👍  For now I just wrote simple page just to have new functionalities documented at all